README.md

```diff
@@ -144,25 +144,23 @@ slow resources. But it is by no means a magic wand that solves all your latency
 problems by being added to your `Gemfile`. This section describes the types of
 problems Semian solves.
 
-If your server isn't single threaded (e.g. Resque and Unicorn) or evented these
-problems are less immediate. However, still recommended at scale.
+If your application is not multithreaded or evented (e.g. Resque or Unicorn)
+these problems are not as pressing. You can still get use out of Semian however.
 
 ### Real World Example
 
-This is better illustrated with a real world example from Shopify. If you are
-browsing a store and you are signed in, Shopify stores your session in a session
-store. If the session store becomes unavailable, the data driver will start
-throwing driver errors. We rescue these exceptions and simply disable all
-customer sign in functionality on the store until the session store is back
-online.
+This is better illustrated with a real world example from Shopify. When you are
+browsing a store while signed in, Shopify stores your session in Redis.
+If Redis becomes unavailable, the driver will start throwing exceptions.
+We rescue these exceptions and simply disable all customer sign in functionality
+on the store until Redis is back online.
```

Comment:
I changed this to just use "Redis" instead of "store/session store" because it was a bit verbose